app/facades/movie_facade.rb

```diff
@@ -1,14 +1,21 @@
 class MovieFacade
-  def initialize 
+  attr_reader :keyword
+  def initialize(keyword = nil)
+    @keyword = keyword
   end
 
   def movies
-    service = TMDBService.new
-    data = service.search_movies
+    service = MovieDatabaseService.new
 
-    @movies = data[:results].map do |movie_data|
+    if @keyword.nil?
+      data = service.top_rated_movies
+    else
+      data = service.search_movies(@keyword)
+    end
+
+    data[:results].map do |movie_data|
```

Comment:
Great logic here to ensure that our routes are directing to the correct API endpoint.